src/api/app/controllers/webui/reports_controller.rb

```diff
@@ -10,7 +10,14 @@ def create
     @link_id = params[:link_id]
 
     if @report.save
-      flash[:success] = "#{@report.reportable_type} reported successfully"
+      if @report.reportable_type == 'Comment' && params[:report_comment_author].present?
+        @user.submitted_reports.create!(report_params.merge(reportable_id: @report.reportable.user_id,
```

Comment:
This can create multiple reports for the same user, as someone can report multiple comments of a user, while reporting this user as the same time over and over again.

Is it an issue? I don't think so. What do you think?

Reply:
I wouldn't think about this from the can/can not angle. What do you think make sense?

Reply:
So lets you delete and report 5 comments of a user. That would trigger the creation of 5 Event::ReportForComment and generates 5 notifications. The created reports are not an issue I would say, but the flood of notifications might be annoying. So maybe the notifications should somehow be merged

Reply:
Ah, I guess this is already done by the `OutdatedNotificationsFinder::Report`

Reply:
> I wouldn't think about this from the can/can not angle. What do you think make sense?

I would go with this implementation. The UX for reporters is much better.